src/components/ToolbarButton/ToolbarButton.module.scss

```diff
@@ -57,6 +57,12 @@
       fill: #fff;
     }
   }
+
+  &:disabled {
+    border: 1px solid #6f7070;
+    color: #fff;
+    background: linear-gradient(to bottom, #949494 0, #878a8a 100%);
+  }
```

Comment:
This might be fine, but the usual way of styling disabled elements in H5P is something like this: 
![image](https://user-images.githubusercontent.com/9085189/148089298-4d549afb-fce8-489a-bb60-12ae7428dfe8.png)

I suggest using a slightly darker color (#6C6D6D instead of #707171) because the latter does not have a high enough contrast against the background color.

```suggestion
  &:disabled {
    background-color: #f5f5f5;
    border-color: #eee;
    color: #6C6D6D;
    cursor: not-allowed;
  }
```

Reply:
👍Thanks, I will look into default styling for elements in h5p for color choices and design in general if similar task arises in future.